lizard_auth_server/templates/lizard_auth_server/confirm_deletion_userconsent.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'lizard_auth_server/base.html' %}
+
+{% block head_title %}
+{{ instance.title }} | {{ block.super }}
+{% endblock head_title %}
+
+{% block container %}
+
+<div class="col-sm-6 col-sm-offset-3">
+
+    <form method="post" action=""> {% csrf_token %}
+        <p>Do you want to delete: "{{ userconsent }}"?</p>
```

Comment:
TODO: nog even naar de teksten kijken. Wat is het duidelijkst. En we moeten er nog even vertaal-tags omheen zetten. Dat zal ik maandag wel even laten zien.

Misschien alvast even kijken of pycharm iets kan met die ".po" files die ook ergens in dit PR staan.

Reply:
Vertaal-tags gedaan. .po files nog niet aangepast.